lib/ember-test-helpers/abstract-test-module.js

```diff
@@ -95,11 +95,15 @@ export default Klass.extend({
   },
 
   setupTestElements() {
-    if (!document.querySelector('#ember-testing')) {
+    let testEl = document.querySelector('#ember-testing');
+    if (!testEl) {
       let element = document.createElement('div');
       element.setAttribute('id', 'ember-testing');
 
       document.body.appendChild(element);
+      this.fixture = '';
+    } else {
+      this.fixture = testEl.innerHTML;
```

Comment:
I can change the name. For initialization, I originally tried it in the constructor, but [`test-module` overrides `init` and doesn't call `_super`](https://github.com/switchfly/ember-test-helpers/blob/77f9a53da9d8c19a85b3122788caadbcc59274c2/lib/ember-test-helpers/test-module.js#L8-L44) so it never gets set in that scenario. I could add it in both locations, but I figured this was cleaner. Does cause the shape to morph, but figured since this was just a testing environment that it's not a huge deal.